src/Nethermind/Nethermind.State/Proofs/ValidatorExitsTrie.cs

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
```

Comment:
For this class for withdrawals we have matching test class:
WithdrawalsTrie ->WithdrawalsTrieTests

Please apply the same for deposits and exits